plugins/cohere/test-fixtures/config

```diff
@@ -0,0 +1 @@
+{"CurrentURL": "https://api.os.cohere.ai","Contexts": {"https://api.os.cohere.ai": {"JWT": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTk2MDYzODksImlhdCI6MTY4ODA3MDM4OSwidXNlcl9pZCI6IjdhZjBiYjFiLTU2MDQtNDI0Ny1hOWVhLWI0ZWI2ZjMzMjBiMCJ9.hCNIQcnrR371wsafGHj_xqnL9-OQbBGySN4Sfp6_wpI","Email": "wendy@appleseed.com"}}}
```

Comment:
Is this an example value for the JWT? Let's make sure it's clear that this is the case, e.g. by ending the string with "EXAMPLE".